### PR TITLE
8Ball Command

### PR DESCRIPTION
**Pull request by Shadwz17**
Created an 8Ball command for giving 'Grace' some fun commands to mess with like this one, where you ask something and Grace will answer you randomly.

### CODE REVIEW

**Comment by Shadwz17** [comment]
This is how it works.

File: bot/extensions/8ball.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+class eightball(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='eightball', help="Ask a question and be answered by Grace", aliases=['8ball'])
+    async def eightball(self, ctx):
+
+        responses = ['Hell no.', 'Prolly not.', 'Idk bro.',
```
Comment (PenguinBoi12) [inline bot/extensions/8ball.py:15]: This is a lot of hard-coded data. For now it's accepted but soon I will have added a configuration system for extensions. The purpose of this config system will be to allow each extension, like this, to have their own configurations and list of data like this.

For now, it's not done so just keep in mind that if I'm done implementing the extension config system before this PR is approved, I will ask you bring the changes for that. If it's approved before, I will take the responsibility upon me to make the changes.

**Review by PenguinBoi12 — CHANGES_REQUESTED** [review]
First of all, good job. You did some nice work. However, there is a few change to make but luckily nothing major. 

Again, good job and have a good day.

Ps. Sorry it took so much time to review your PR.

**Comment by Brokenhammer72** [comment]
Okay It would be good if we could make a a channel for this and whenever there is a message in the channel it grace will reply in that channel  also putting ::eightball everytime is boring . if we have a channel we can add a slowmode so people dont spam questions

File: bot/extensions/8ball.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                     'Yes!', 'No!', 'Signs a point to Yes!',
+                     'Reply Hazy, Try again.', 'IDK m8 try again.',
+                     'Better not tell you know.', 'Cannot predict now.',
+                     'Concentrate and ask again.', "Don't Count on it.",
+                     'My reply is No.', 'My sources say No.',
+                     'Outlook not so good.', 'Very Doubtful']
+
+        answer = Embed(
+            title="Grace says: ",
+            Color=0x00ff00,
```
[the remark this answers is not in the document]
Comment (Shadwz17) [reply bot/extensions/8ball.py]: Added the **bot.default_color** as main color for the embed.

File: bot/extensions/8ball.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import discord
+from discord.colour import Color
+from discord.ext.commands import Cog, command
+from discord import Embed
+import random
+
+
+class eightball(Cog):
```
[the remark this answers is not in the document]
Comment (Shadwz17) [reply bot/extensions/8ball.py]: Updated the names of both the class and the name of the file.

File: bot/extensions/8ball.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='eightball', help="Ask a question and be answered by Grace", aliases=['8ball'])
```
[the remark this answers is not in the document]
Comment (Shadwz17) [reply bot/extensions/8ball.py]: Added the usage and moved the aliases after the **name=... ,**

File: bot/extensions/eight_ball.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import discord
+from discord.ext.commands.cooldowns import BucketType
+from discord.ext.commands import Cog, command, cooldown
+from discord import Embed
+import random
+
+
+class EightBall(Cog):
```
Comment (rachit1601) [inline bot/extensions/eight_ball.py:8]: Add description to the cog also for better help menu

Comment (PenguinBoi12) [reply bot/extensions/eight_ball.py:8]: It's optional but indeed it could be a nice addition.

**Review by rachit1601 — APPROVED** [review]

**Review by rachit1601 — APPROVED** [review]

**Review by PenguinBoi12 — APPROVED** [review]
I brought the changes for the description and usage because they were really minor changes. So congratulation for you first approved PR. I hope you will continue to contribute to the bot.

**Pull request merged**